data/json/items/tools.json

```diff
@@ -1,4 +1,4 @@
-[
+﻿[
```

Comment:
If you didn't tell Notepad to save it as "all files" or otherwise not mess with the format, it probably did mess you up.

Notepad++ is used by just about everyone (including me!) who handles DDA code on windows; if there's some reason you'd rather not use it, let us know and we'll see about alternates?